README.md

```diff
@@ -18,7 +18,7 @@ money_column expects a decimal 8,3 database field.
 
 ## Installation
 
-    gem 'money', github: 'shopify/money'
+    gem 'shopify-money', require: 'money'
```

Comment:
if you want you can even rename the file names and constant and just do a `Money = ShopifyMoney` in `lib/shopify-money.rb`. But not required to do in this PR.